0051-helium-dao.md

```diff
@@ -50,6 +50,27 @@ There are three key drivers for proposing that DNP specific calculations are don
 
 This proposal impacts all current and future stakeholders in the Helium Community.
 
+# **Implementation**
+
+HIP 51 is to be implemented in three phases, each of which is contingent upon approval via governance across three separate voting procedures. Note that each vote, and therefore phase, must necessarily have code attached for consideration.
+
+The phases are defined as follows:
+
+1. Phase I (June 9, 2022) : As per HIP 53: Helium Mobile subDAO, MOBILE token is issued on the working Helium blockchain. 
```

Comment:
```suggestion
1. Phase I (Q2 2022) : As per HIP 53: Helium Mobile subDAO, MOBILE token is issued on the existing Helium blockchain. This will immediately allow any online 5G radio to mine MOBILE tokens.
```

Reply:
be more explicit about what this enables and also generalize the timeline a bit.